certora/specs/Health.spec

```
@@ -74,7 +48,7 @@ filtered { f -> !f.isView }
 
     mathint collateralAfter = collateral(id, user);
 
-    assert !priceChanged => collateralAfter >= collateralBefore;
+    assert collateralAfter >= collateralBefore;
```

optional and remove line 49
```suggestion
    assert collateralAfter >= collateral(id, user);
```